fatiando/mesher/geometry.py

```diff
@@ -92,6 +94,57 @@ def x(self):
     def y(self):
         return self.vertices[:, 1]
 
+    @property
+    def orientation(self):
+        area = self.get_area(absolute=False)
+        if area < 0:
+            return "CW"
+        else:
+            return "CCW"
+
+    def set_clockwise(self):
+        self.set_orientation("CW")
+
+    def set_orientation(self, orientation):
+        if orientation not in ["CW", "CWW"]:
+            raise ValueError("Orientation must be 'CW' or 'CCW'")
+        current = self.orientation
+        if orientation != current:
+            self._vertices = self._vertices[::-1]
```

Comment:
A more Pythonic way of implementing this method would be as a property `setter` so that we could do `polygon.orientation = 'clockwise'` to set it:

    @orientation.setter
    def orientation(self, new_orientation):

The rest stays the same.